### ISSUE TITLE
【EESS】New submissions for Mon, 10 Jul 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#393]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Empirical Analysis of a Segmentation Foundation Model in Prostate  Imaging
 - **Authors:** Heejong Kim, Victor Ion Butoi, Adrian V. Dalca, Mert R. Sabuncu
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2307.03266
 - **Pdf link:** https://arxiv.org/pdf/2307.03266
 - **Abstract**
 Most state-of-the-art techniques for medical image segmentation rely on deep-learning models. These models, however, are often trained on narrowly-defined tasks in a supervised fashion, which requires expensive labeled datasets. Recent advances in several machine learning domains, such as natural language generation have demonstrated the feasibility and utility of building foundation models that can be customized for various downstream tasks with little to no labeled data. This likely represents a paradigm shift for medical imaging, where we expect that foundation models may shape the future of the field. In this paper, we consider a recently developed foundation model for medical image segmentation, UniverSeg. We conduct an empirical evaluation study in the context of prostate imaging and compare it against the conventional approach of training a task-specific segmentation model. Our results and discussion highlight several important factors that will likely be important in the development and adoption of foundation models for medical image segmentation.
### Equivariant Spherical CNN for Data Efficient and High-Performance  Medical Image Processing
 - **Authors:** Amirreza Hashemi, Yuemeng Feng, Hamid Sabet
 - **Subjects:** Image and Video Processing (eess.IV); Machine Learning (cs.LG); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2307.03298
 - **Pdf link:** https://arxiv.org/pdf/2307.03298
 - **Abstract**
 This work highlights the significance of equivariant networks as efficient and high-performance approaches for tomography applications. Our study builds upon the limitations of Convolutional Neural Networks (CNNs), which have shown promise in post-processing various medical imaging systems. However, the efficiency of conventional CNNs heavily relies on an undiminished and proper training set. To tackle this issue, in this study, we introduce an equivariant network, aiming to reduce CNN's dependency on specific training sets. We evaluate the efficacy of equivariant CNNs on spherical signals for tomographic medical imaging problems. Our results demonstrate superior quality and computational efficiency of spherical CNNs (SCNNs) in denoising and reconstructing benchmark problems. Furthermore, we propose a novel approach to employ SCNNs as a complement to conventional image reconstruction tools, enhancing the outcomes while reducing reliance on the training set. Across all cases, we observe a significant decrease in computational costs while maintaining the same or higher quality of image processing using SCNNs compared to CNNs. Additionally, we explore the potential of this network for broader tomography applications, particularly those requiring omnidirectional representation.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Equivariant Spherical CNN for Data Efficient and High-Performance  Medical Image Processing
 - **Authors:** Amirreza Hashemi, Yuemeng Feng, Hamid Sabet
 - **Subjects:** Image and Video Processing (eess.IV); Machine Learning (cs.LG); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2307.03298
 - **Pdf link:** https://arxiv.org/pdf/2307.03298
 - **Abstract**
 This work highlights the significance of equivariant networks as efficient and high-performance approaches for tomography applications. Our study builds upon the limitations of Convolutional Neural Networks (CNNs), which have shown promise in post-processing various medical imaging systems. However, the efficiency of conventional CNNs heavily relies on an undiminished and proper training set. To tackle this issue, in this study, we introduce an equivariant network, aiming to reduce CNN's dependency on specific training sets. We evaluate the efficacy of equivariant CNNs on spherical signals for tomographic medical imaging problems. Our results demonstrate superior quality and computational efficiency of spherical CNNs (SCNNs) in denoising and reconstructing benchmark problems. Furthermore, we propose a novel approach to employ SCNNs as a complement to conventional image reconstruction tools, enhancing the outcomes while reducing reliance on the training set. Across all cases, we observe a significant decrease in computational costs while maintaining the same or higher quality of image processing using SCNNs compared to CNNs. Additionally, we explore the potential of this network for broader tomography applications, particularly those requiring omnidirectional representation.
### Waveform-Domain Adaptive Matched Filtering: A Novel Approach to  Suppressing Interrupted-Sampling Repeater Jamming
 - **Authors:** Hanning Su, Qinglong Bao, Jiameng Pan, Fucheng Guo, Weidong Hu
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.03368
 - **Pdf link:** https://arxiv.org/pdf/2307.03368
 - **Abstract**
 The inadequate adaptability to flexible interference scenarios remains an unresolved challenge in the majority of techniques utilized for mitigating interrupted-sampling repeater jamming (ISRJ). Matched filtering system based methods is desirable to incorporate anti-ISRJ measures based on prior ISRJ modeling, either preceding or succeeding the matched filtering. Due to the partial matching nature of ISRJ, its characteristics are revealed during the process of matched filtering. Therefore, this paper introduces an extended domain called the waveform domain within the matched filtering process. On this domain, a novel matched filtering model, known as the waveform-domain adaptive matched filtering (WD-AMF), is established to tackle the problem of ISRJ suppression without relying on a pre-existing ISRJ model. The output of the WD-AMF encompasses an adaptive filtering term and a compensation term. The adaptive filtering term encompasses the adaptive integration outcomes in the waveform domain, which are determined by an adaptive weighted function. This function, akin to a collection of bandpass filters, decomposes the integrated function into multiple components, some of which contain interference while others do not. The compensation term adheres to an integrated guideline for discerning the presence of signal components or noise within the integrated function. The integration results are then concatenated to reconstruct a compensated matched filter signal output. Simulations are conducted to showcase the exceptional capability of the proposed method in suppressing ISRJ in diverse interference scenarios, even in the absence of a pre-existing ISRJ model.
### Using electrical impedance spectroscopy to identify equivalent circuit  models of lubricated contacts with complex geometry: in-situ application to  mini traction machine
 - **Authors:** Min Yu, Jie Zhang, Arndt Joedicke, Tom Reddyhoff
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.03668
 - **Pdf link:** https://arxiv.org/pdf/2307.03668
 - **Abstract**
 Electrical contact resistance or capacitance as measured between a lubricated contact has been used in tribometers, partially reflecting the lubrication condition. In contrast, the electrical impedance provides rich information of magnitude and phase, which can be interpreted using equivalent circuit models, enabling more comprehensive measurements, including the variation of lubricant film thickness and the asperity (metal to metal) contact area. An accurate circuit model of the lubricated contact is critical as needed for the electrical impedance analysis. However, existing circuit models are hand derived and suited to interfaces with simple geometry, such as parallel plates, concentric and eccentric cylinders. Circuit model identification of lubricated contacts with complex geometry is challenging. This work takes the ball-on-disc lubricated contact in a Mini Traction Machine (MTM) as an example, where screws on the ball, grooves on the disc, and contact close to the disc edge make the overall interface geometry complicated. The electrical impedance spectroscopy (EIS) is used to capture its frequency response, with a group of load, speed, and temperature varied and tested separately. The results enable an identification of equivalent circuit models by fitting parallel resistor-capacitor models, the dependence on the oil film thickness is further calibrated using a high-accuracy optical interferometry, which is operated under the same lubrication condition as in the MTM. Overall, the proposed method is applicable to general lubricated interfaces for the identification of equivalent circuit models, which in turn facilitates in-situ tribo-contacts with electric impedance measurement of oil film thickness. It does not need transparent materials as optical techniques do, or structural modifications for piezoelectric sensor mounting as ultrasound techniques do.
